internal/pkg/manifest/validate_env.go

```diff
@@ -235,3 +241,15 @@ func (c environmentConfig) validateInternalALBSubnets() error {
 	}
 	return nil
 }
+
+func areSetsEqual[T comparable](a map[T]struct{}, b map[T]struct{}) bool {
```

Comment:
one of these days, we should add a `sets` pkg like how we have a `graphs` pkg

Reply:
this'll exist soon (hopefully)! https://pkg.go.dev/golang.org/x/exp/maps#Equal